### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Nashaath**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,112 +29,124 @@ Once you you have all the prerequisites set up, you can follow the instructions
...
-7. Expand Local & Outbound Authentication Configuration.
-8. Select Advanced Configuration
+5. Click **Finish**. This displays the **Users** screen that lists the user you created.
+6. Click **User Profile** on the user record that you created.
+7. In the user profile specify the following:
+   * Enter the **First Name** and **Email**.
+   * Enter the **token_id** that is allocated to the user.
 
-![18](images/18.png "18")
+    ![14](images/1.png "14")
```
Comment (wijith7) [inline README.md]: can you change this to    (images/14.png "14")

File: README.md [hunk 2]
```diff
@@ -29,112 +29,124 @@ Once you you have all the prerequisites set up, you can follow the instructions
...
+      * **User Store Domain Name**: PRIMARY
+      * **Mapped Attribute**: displayName
+      >> If the displayName attribute is already in use, you need to map another attribute.
 
     ![10](images/10.png "10")
 
-4. Click **Add**.
-5. Edit token_id from the list as follows.
+5. Click **Add**. This adds token_id under the list of **Available Claims for http://wso2.org/claims**.
+6. Click **Edit** to edit the token_id claim, and then do the following changes:
```
Comment (wijith7) [inline README.md]: we don't need this step

File: README.md [hunk 3]
```diff
@@ -29,112 +29,124 @@ Once you you have all the prerequisites set up, you can follow the instructions
...
+   * Specify the following as the **Regular Expression**:
+        ^[a-fA-F0-9]{3} [0-9]{6}$
+        
+   * Select **Supported by Default**.
```
Comment (wijith7) [inline README.md]: add this to step 4

File: README.md [hunk 4]
```diff
@@ -29,112 +29,124 @@ Once you you have all the prerequisites set up, you can follow the instructions
...
+   
+   ![11](images/11.png "11")
```
Comment (wijith7) [inline README.md]: remove this

File: README.md [hunk 5]
```diff
@@ -29,112 +29,124 @@ Once you you have all the prerequisites set up, you can follow the instructions
...
 
-    ![14](images/1.png "14")
+![18](images/18.png "18")
+
+9. Click **Add Authentication Step**.
+10.Under **Local Authenticators** in **Step 1**, select **basic** and click **Add Authenticator**.
+11.Click **Add Authentication Step**.
+12.Under **Local Authenticators** in **Step 2**, select **casque_auth** and click **Add Authenticator**.
+13.Select **Use subject identifier from this step** under **Step 1**.
+14.Select **Use attributes from this step** under **Step 2**.
```
Comment (wijith7) [inline README.md]: put space with number

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**